CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Added
 
 - `show-config` subcommand to display currently used configuration
+- `--hex-format` flag has been added
+
+#### Removed
+- `-i` for `--int-format` is removed
```

Comment:
Let's make this message more explicit - for a moment I thought the `--int-format` flag was removed altogether, and not only the abbreviation :D

Reply:
haha, fixed it!